app/classifier/tutorial.jsx

```diff
@@ -290,19 +306,20 @@ export default class Tutorial extends React.Component {
           </button>
 
             <span className="step-through-pips">
-              {allSteps.map(thisStep =>
+              {allSteps.map(thisStep => (
                 <label key={thisStep} className="step-through-pip" title={`Step ${thisStep + 1}`}>
                   <input
                     type="radio"
+                    name="tutorial-step"
                     className="step-through-pip-input"
-                    aria-label={`Step ${thisStep + 1} of ${totalSteps}`}
+                    aria-label={`Tutorial step ${thisStep + 1} of ${totalSteps}`}
```

Comment:
Ideally, this text would be localised but we don't have any multi-lingual audio projects at the moment.